.github/workflows/test-suite.yml

```diff
@@ -159,6 +159,28 @@ jobs:
     - name: Show cache stats
       if: env.SELF_HOSTED_RUNNERS == 'true'
       run: sccache --show-stats
+  http-api-tests:
```

Comment:
CI is failing because this job hasn't been added to `test-suite-success`:

https://github.com/sigp/lighthouse/blob/e0c1f27e1303ba0b4f86585241e7f88ccb4a9bbb/.github/workflows/test-suite.yml#L469-L481